.env

```diff
@@ -1,4 +1,10 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost.bbc.com:7080
```

Comment:
This file should not be committed

https://github.com/bbc/simorgh/blob/b33a3ce76526d88b7e4dea762379ef0046d22f6f/.env#L3